### DISCUSSION TITLE
Is this package also update the firebase email verification status that (user.emailVerified;)

### DISCUSSION THREAD

**Discussion by champ96k** [saran-surya/email_auth#72] [Q&A]
Is this package also update the firebase email verification status that user.emailVerified flag @saran-surya 

```
final user = FirebaseAuth.instance.currentUser;
if (user != null) {

    // Check if user's email is verified [This flag is also updated. to true ?]
    final emailVerified = user.emailVerified;
}

```

Reference: https://firebase.flutter.dev/docs/auth/manage-users

**Comment by saran-surya** [comment]
Hi @champ96k,
As of now there is no link with firebase regarding this. Also there is a discussion with Heroku regarding hosting. Once that is confirmed, V2 wil be released, will surely try to consider this for an implementation. Also firebase already provides a auth verification system, so it it's not reinventing the wheel, then surely it is a go.

Thanks

**Reply by champ96k** [reply]
Hi @saran-surya ,

Thanks for the response, yes firebase also provides email verification, but the thing is that on firebase you need to verify email through a link, not through OTP.

I point in firebase to verify email through OTP is still missing. so this is a great time to launch your ship 🚀.

**Reply by saran-surya** [reply]
Hi @champ96k ,
Will surely look, because offering a free service might cause some to use it for unwanted purposes. Will be checking regarding it.
